src/Abstractions/TaskName.cs

```diff
@@ -33,7 +32,8 @@ public TaskName(string name, string? version = null)
     /// Gets the version of the task.
     /// </summary>
     /// <remarks>
-    /// Task versions are currently experimental and their role may change over time.
+    /// Task versions is currently locked to <see cref="string.Empty" /> as it is not yet integrated into task
+    /// identification. This is being left here as we intend to support it soon.
     /// </remarks>
     public string Version { get; }
```

Comment:
Would it make sense to put an `[Obsolete]` attribute on this field or do we still reference it in some places?

Reply:
We do reference it. I don't want to add `[Obsolete]` because I don't want to discourage logic from taking an actual dependency on it one day not being `""`.